GeoJSON/geojson-encoding-rule.md

```diff
@@ -0,0 +1,270 @@
+
+* [GeoJSON - IETF RFC 7946](https://tools.ietf.org/html/rfc7946)
+* [D2.7 INSPIRE DRAFTING TEAM “DATA SPECIFICATIONS.” D2.7: Guidelines for the encoding of spatial data, Version 3.3](http://inspire.ec.europa.eu/documents/guidelines-encoding-spatial-data)
+
+## Terms and Definitions
+
+Terms and Definitions can be found in the [Glossary](../glossary.md) document.
+
+## Schema Conversion Rules
+
+INSPIRE defines the conceptual model using UML.
+
+The default encoding rule maps this UML model to XML Schema. For JSON, [JSON Schema](https://json-schema.org/) can be used to perform simple validation on JSON documents.  
+
+NOTE At this point, [JSON Schema](https://json-schema.org/) is not used in any of the targeted GIS tools. We thus do not yet use it normatively, but to provide a formal definition of the encoded data structures, and to help develop executable test suites in future versions. 
+
+In this encoding rule, we take a two-step approach, where we apply model transformations on the level of the conceptual model. This model can then be encoded in simple GeoJSON using the general schema and instance conversion rules laid out in the next sections.
+
+### Types
+
+#### Feature types 
+
+All types that have the stereotype `<<featureType>>` are converted to GeoJSON objects. Typenames remain as they are. 
+
+#### Data Types
+
+All data types are mapped to regular JSON objects, where properties of the type are directly added to the root object instead of being added to the `properties` subobject. Their typenames remain as they are.
+
+#### ISO 19103 - Basic types
+
+All property types are transformed to the simple types that JSON knows about: Number, String, Boolean and Object. The exact mapping from the UML model to the JSON datatypes is outlined in the following table:
+
+| UML Model property type | JSON datatype | Conversion Notes | 
+| ------ | ----- | ----- |
+| CharacterString | string |  |
+| LocalisedCharacterString | string | `LanguageCode` is added as a separate property. |
+| Boolean | boolean |  |
+| Integer | integer |  |
+| Real | number |  |
+| Decimal | number |  |
+| DateTime | string | The string must follow the ISO 8601 format, including timezone information (e.g. `2008-10-31T15:07:38-05:00`). |
+| Date | string | The string most follow the format `yyyy-mm-dd`. |
+| Length | double | `uom` is added as a separate property. |
+| Measure | double | `uom` is added as a separate property. |
+| URI | string |  |
+| URL | string |  |
+
+Any other UML Model property type are to be mapped to `string`, with specific rules being defined on a case-by-case basis in each theme profile.
+
+#### ISO 19107 - Geometry types
+
+ISO 19107 defines a set of Geometry types, which need to be mapped to the types available in GeoJSON. Note that not all types can be mapped to GeoJSON; if an data set requires such a type, it cannot use this encoding rule as an alternative encoding rule.
```

Comment:
Make the note a separate paragraph. 

Is there a way to mark this as a "warning" (with an icon)?